framework/include/mesh/MooseMesh.h

```diff
@@ -739,6 +741,11 @@ class MooseMesh : public MooseObject, public Restartable, public PerfGraphInterf
   void
   changeBoundaryId(const boundary_id_type old_id, const boundary_id_type new_id, bool delete_prev);
 
+  static void changeBoundaryId(MeshBase & mesh,
```

Comment:
Are you calling any of these new methods anywhere, or are you planning on doing this in a follow up PR?
Can you put doc strings in for these new methods or use the "///@{ ///@}" syntax to group them with the existing member methods for Doxygen?

Reply:
> Are you calling any of these new methods anywhere, or are you planning on doing this in a follow up PR?

I call this one from the original method 😄 But yea I wrote these for the `SmearedPelletMeshGenerator` that I'm writing for BISON. I could probably drum up a framework test.